core/test/config_test.jl

```diff
@@ -82,3 +82,9 @@ end
     @test Ribasim.snake_case("ABCdef") == "a_b_cdef"
     @test Ribasim.snake_case("snake_case") == "snake_case"
 end
+
+@testitem "camel_case" begin
```

Comment:
Also (or only?) test for the symbol input, as indicated by codecov

Reply:
Good idea to add the symbol input tests. Since this function also has an overloading that take strings, removing the string tests may trigger the codecov indication again. We should keep it